psql2mysql/__init__.py

```diff
@@ -223,20 +243,20 @@ def migrate(self):
             self.target_db.clearTable(table)
 
         for table in source_tables:
-            LOG.info("Migrating table: '%s'" % table.name)
-            self.setupTypeDecorators(table, target_tables[table.name])
-            if table.name not in target_tables:
-                raise Exception(
-                    "Table '%s' does not exist in target database" %
-                    table.name)
-
             # skip the schema migration related tables
             # FIXME: Should we put this into a config setting
             # (e.g. --skiptables?)
             if (table.name == "migrate_version" or
                     table.name.startswith("alembic_")):
                 continue
 
+            LOG.info("Migrating table: '%s'" % table.name)
```

Comment:
Why did you move this block?

Reply:
Actually I moved the skipping migrate and alembic tables up, because if were skipping them we shouldn't require the target to actually have the tables sitting there empty.

Also the empty mysql neutron db I used didn't have one of these tables (which I got from another source). so it was also causing me issues.

Reply:
Actually, we only skip those tables because it's assumed that the existing mysql database already has them, because the respecitve "db-manage" tool was already run against it. (So they should be in the very same state as the source db already). I think throwing an error if the source has them but the target doesn't is still a good idea.